securethenews/blog/models.py

```diff
@@ -56,6 +56,6 @@ def posts(self):
         posts = BlogPost.objects.live().descendant_of(self)
 
         # Order by most recent date first
-        posts = posts.order_by('-date')
+        posts = posts.order_by('-id')
```

Comment:
That's a good description of a real edge case, @garrettr, but the proposed changes are generally applicable enough that this makes a good candidate for merge. Especially with the tests! 

Given the low rate of publishing on the site, I think we can live with the edge case, especially since we're totally aware of it. If it becomes an issue we can use solution 1) that you described in #97 and write a migration for it.